shippable.yml

```diff
@@ -47,6 +47,8 @@ build:
     - extra_directories=($SHIPPABLE_REPO_DIR/build/*scripts*)
     - extra_path=$(printf "%s:" "${extra_directories[@]}")
     - export PATH="${extra_path}${PATH}"
+    # check OpenBLAS version
+    - python tools/openblas_support.py --check_version 0.3.5
```

Comment:
I'll try to bump this to 0.3.7.dev in a separate PR-- I need to log in to gcc compile farm on another machine to get that updated though